ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseEventMapper.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.cmc.ccd.mapper;
+
+import uk.gov.hmcts.cmc.ccd.domain.CaseEvent;
+import uk.gov.hmcts.cmc.domain.models.ClaimDocumentType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.*;
+import static uk.gov.hmcts.cmc.domain.models.ClaimDocumentType.*;
+
+public class CaseEventMapper {
+
+    private static final Map<ClaimDocumentType, CaseEvent> claimDocumentTypeMap;
+
+    static {
+        claimDocumentTypeMap = new HashMap<>();
+        claimDocumentTypeMap.put(SEALED_CLAIM, SEALED_CLAIM_UPLOAD);
+        claimDocumentTypeMap.put(CLAIM_ISSUE_RECEIPT, CLAIM_ISSUE_RECEIPT_UPLOAD);
+        claimDocumentTypeMap.put(DEFENDANT_RESPONSE_RECEIPT, DEFENDANT_RESPONSE_UPLOAD);
+        claimDocumentTypeMap.put(CCJ_REQUEST, CCJ_REQUEST_UPLOAD);
+        claimDocumentTypeMap.put(SETTLEMENT_AGREEMENT, SETTLEMENT_AGREEMENT_UPLOAD);
+        claimDocumentTypeMap.put(DEFENDANT_PIN_LETTER, DEFENDANT_PIN_LETTER_UPLOAD);
+    }
+
+    public static CaseEvent map(ClaimDocumentType type) {
+
```

Comment:
Dont use default. if the type is not there, throw exception.

Reply:
As discussed. For the document type to event mapping:
1. It was the default before we added these mappings, so it's simply doing what it did before.
2. By throw an exception, we're effectively adding functionality and an additional step for support to deal with
3. We don't want the document upload to fail just because we can't find a mapping for the event type
4. In 3 the case worker will still be able to view the document in CCD.
5. The document will still get upload, albeit possibly not the correct event type. I think this is okay because it probably just means that no specific event type is defined for the document type and it would just be added as a future requirement